temporaltest/options.go

```diff
@@ -33,6 +34,12 @@ func WithBaseClientOptions(o client.Options) TestServerOption {
 	})
 }
 
+func WithBaseWorkerOptions(o worker.Options) TestServerOption {
+	return newApplyFuncContainer(func(server *TestServer) {
+		server.defaultWorkerOptions = o
```

Comment:
Are there any fields on the base worker options that always need to be overridden?

Reply:
I'll move the worker.FailWorkflow up to the defaultOptions